### DISCUSSION TITLE
A11y wire:navigate doesn't work with keyboard input

### DISCUSSION THREAD

**Discussion by derekcavaliero** [livewire/livewire#6252] [1. Help]
Is there a modifier that can be added to the `wire:navigate` attribute to allow for activating a link when the user hits enter on a focused link on their keyboard?

I was digging around through the livewire.js file and it looks like there are bindings for "keydown" but they look to be specific to Alpines focus plugin.

**Comment by WyattCast44** [comment]
I also ran into this issue, I agree that it looks like a Alpine navigate package implementation area. 

One potential solution, 

- When an anchor is focused, add an event listener to the element for a keydown event (specifically the enter key)
- If the enter key is pressed while the anchor is focused, execute the normal wire:navigate flow
- When the anchor is unfocused, remove the event listener

What do you think @joshhanley @calebporzio , I can put together a PR in the Alpine repo if this something you think you would support?

**Comment by derekcavaliero** [comment]
@WyattCast44 throwing this here as a temporary solution, but you can add something like this into a global JS file which seems to work pretty well.

```javascript
document.addEventListener('keydown', (e) => {
  
  if (!e.target.hasAttribute('wire:navigate'))
    return;

  if (e.key.toLowerCase() == 'enter')
    Alpine.navigate(e.target.href);

});
```

**Comment by PatrickJoannisse** [comment]
I closed my discussion on this because this one was created first but I provided barebone code sample in mine. https://github.com/livewire/livewire/discussions/6585

**Comment by gehrisandro** [comment]
My PR addresses this issue: https://github.com/livewire/livewire/pull/6673
